### ISSUE TITLE
Light theme having a dark code editor is not accessible for those who need light theme for visual accessibility

### ISSUE THREAD

**Issue by yaycupcake** [summitech/gitexplorer#113]
It would be helpful to have a theme option with both a light site theme as well as a light code editor theme. A lot of folks have trouble reading or looking at high contrast dark themes, even for code editors, including people with astigmatism or who experience migraines. This would be a fantastic accessibility fix, if you could implement it.
